packages/demo/src/searchBar/StandaloneSearchBar.tsx

```diff
@@ -0,0 +1,88 @@
+import '@demo-styling/standaloneSearchBar.scss';
+
+import {buildSearchBox, SearchBox as HeadlessSearchBox} from '@coveo/headless';
+import {FunctionComponent, useContext, useEffect, useState} from 'react';
+import * as React from 'react';
+import {Button, ListBox, Svg} from 'react-vapor';
+
+import classNames from 'classnames';
+import {EngineContext} from './engine/EngineContext';
+
+interface ISearchboxProps {
+    id: string;
+}
+
+const SearchBoxRenderer: FunctionComponent<{
+    id: string;
+    controller: HeadlessSearchBox;
+}> = (props) => {
+    const {id, controller} = props;
+    const [state, setState] = useState(controller.state);
+
+    useEffect(() => controller.subscribe(() => setState(controller.state)), []);
+
+    const ClearButton = () => (
+        <button
+            disabled={state.value === ''}
+            className={classNames('clear-button', {'search-not-empty': state.value !== ''})}
+            onClick={() => {
+                controller.updateText('');
+                controller.submit();
+            }}
+        >
+            <Svg svgName="cross" svgClass="icon" />
+        </button>
+    );
+
+    const SearchButton = () => (
+        <Button
+            classes={['search-button']}
+            onClick={() => {
+                controller.submit();
+            }}
+        >
+            <Svg svgName={'search'} className="icon mod-stroke" />
+        </Button>
+    );
+
+    const SearchResultListBox = () => (
+        <>
+            {(state.suggestions.length > 0 || state.isLoadingSuggestions) && (
+                <ListBox
+                    classes={['search-results-container']}
+                    isLoading={state.isLoadingSuggestions}
+                    items={state.suggestions.map((s) => ({
```

Comment:
For now `state.suggestions` is always empty because we need ML to get some events first :see_no_evil: